yang/frr-bfdd.yang

```diff
@@ -169,7 +169,7 @@ module frr-bfdd {
 
     leaf desired-transmission-interval {
       type uint32 {
-          range "10000..60000000";
+          range "10000..4294967000";
```

Comment:
Should we just use `max` instead of manually specifying the number? `"10000..max"`. I know that it won't be possible to set more than 4294967000 using CLI, but it's possible using the API and it doesn't make sense to limit this.

Reply:
changed to max